EDU-3996: Add documentation for how to connect with api keys and multi-region namespaces (#3361)
temporalio/documentation: docs/production-deployment/cloud/account-setup/certificates.mdx

To view the current certificate filters, use the [tcld namespace certificate-filters export](/cloud/tcld/namespace/#export) command.

## Configure clients to use client certificates {#configure-clients}

- [Go SDK](/develop/go/temporal-clients#connect-to-temporal-cloud)
- [Java SDK](/develop/java/temporal-clients#connect-to-temporal-cloud)
- [PHP SDK](/develop/php/temporal-clients#connect-to-a-dev-cluster)
- [Python SDK](/develop/python/temporal-clients#connect-to-temporal-cloud)
- [TypeScript SDK](/develop/typescript/temporal-clients#connect-to-temporal-cloud)
- [.NET SDK](/develop/dotnet/temporal-client#connect-to-temporal-cloud)

### Configure Temporal CLI {#configure-temporal-cli}

> To connect to a Temporal Namespace using the Temporal CLI and certificates specify your certificates and the TLS server name.

Reviewer: Comma

```suggestion
To connect to a Temporal Namespace using the Temporal CLI and certificates, specify your certificates and the TLS server name.
```

Reviewer: Suggestion beyond comma that you might not like, so I did two comments:

```suggestion
To connect to a Temporal Namespace using the Temporal CLI and certificate authentication, specify your credentials and the TLS server name:
```

> ```bash

Reviewer: Quick notes:
- Wrapped is much easier to read.
- Also, `sh` is enough to format correctly for temporal CLI commands vs bash. Both are fine though. Just a "letting you know"

Reviewer: I fold it by hand and put it into a code fence, making sure to indent and use backslashes for line continuation

Author: oh nvm I see what you mean, you want to add wrapped via "\"

> temporal --tls-ca-path <Path to server CA certificate> --tls-cert-path <Path to x509 certificate> --tls-key-path <Path to private certificate key> --tls-server-name <Override for target TLS server name>

Reviewer: ```suggestion
temporal --tls-ca-path <Path to server CA certificate> \
    --tls-cert-path <Path to x509 certificate> \
    --tls-key-path <Path to private certificate key> \
    --tls-server-name <Override for target TLS server name>
```

Reviewer: Also consider whether this would be issued with additional commands and subcommands. If so,  adding placeholders are nice:

```
temporal <command> <subcommand> \
    ...auth stuff...
```
